src/main/kotlin/dev/stalla/dom/DomParsingExtensions.kt

```diff
@@ -176,6 +171,7 @@ internal fun Node.toRssCategoryBuilder(categoryBuilder: RssCategoryBuilder): Rss
  * @return The [ItunesCategory] populated with the DOM node contents.
  */
 @InternalApi
+@Suppress("ReturnCount")
 internal fun Node.toItunesCategory(namespace: FeedNamespace? = null): ItunesCategory? {
     val categoryValue = getAttributeValueByName("text")?.trim() ?: return null
     val category = ItunesCategory.of(categoryValue) ?: return null
```

Comment:
The alternatives I came up with were harder to read and still has too many returns 😅. Also harder to read most times.

Reply:
Suppression is ok here too yep